python_transport/wirepas_gateway/transport_service.py

```diff
@@ -19,6 +19,7 @@
     GatewayState,
     GatewayAPIParsingException,
 )
+from .__about__ import __version__ as transport_version
```

Comment:
I would favor importing it from the package

from Wirepas_gateway import __version__